addons/sale_timesheet/models/sale_order.py

```diff
@@ -151,8 +152,12 @@ class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
     qty_delivered_method = fields.Selection(selection_add=[('timesheet', 'Timesheets')])
-    project_id = fields.Many2one('project.project', 'Generated Project', index=True, copy=False, help="Project generated by the sales order item")
-    task_id = fields.Many2one('project.task', 'Generated Task', index=True, copy=False, help="Task generated by the sales order item")
+    project_id = fields.Many2one(
+        'project.project', 'Generated Project', check_company=True,
+        index=True, copy=False, help="Project generated by the sales order item")
```

Comment:
@jem-odoo is cross-company creation of tasks/timesheets supported from the SO? I know we do (or at least used to do it, not sure now) for odoo.com when e.g. selling a pack in HK but when the consultant is in BE